api/src/main/java/org/openmrs/CurrentUsers/CurrentUsers.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.CurrentUsers;
+
+/**
+ * Utility methods which maintain the list of current users.
+ */
+public class CurrentUsers extends CurrentUsersService {
```

Comment:
I don't really understand why we have this class? Also, Java packages should never start with a capital letter.